specification/app/resource-manager/Microsoft.App/preview/2023-08-01-preview/AppSpaceResources.json

```diff
@@ -0,0 +1,369 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-03-01",
+    "title": "AppSpaceResources API Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/appSpaces/{appSpaceName}": {
+      "get": {
+        "tags": [
+          "AppSpaceResources"
+        ],
+        "summary": "Get the App Resources in a given App Space.",
+        "operationId": "AppSpaces_ListByResourceGroup",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "appSpaceName",
+            "in": "path",
+            "description": "Name of the App Space.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/AppSpaceResourcesCollection"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List App Spaces by resource group": {
+            "$ref": "./examples/AppSpaceResources_ListByAppSpace.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/appSpaces/{appSpaceName}/spaceResources/{spaceResourceName}": {
```

Comment:
the type is App Space Resources - spaces doesn't make sense because the parent type is a space, whereas this type is a space's resource. "resources", likewise, is probably too vague. Would "appSpaceResources" be better?